Gemfile.lock

```diff
@@ -4,7 +4,6 @@ PATH
     app_profiler (0.1.2)
       activesupport (>= 5.2)
       rack
-      railties (>= 5.2)
```

Comment:
This is probably due to https://github.com/Shopify/app_profiler/pull/63 and not the changes here.